appserver/admingui/common/src/main/java/org/glassfish/admingui/common/handlers/RestApiHandlers.java

```diff
@@ -194,6 +194,39 @@ public static void createEntity(HandlerContext handlerCtx) {
     }
 
     /**
+     *
+     * REST-based version of createProxy
+     * @param handlerCtx
+     */
+    @Handler(id = "gf.createEntityResponseOutput",
+            input = {
+                    @HandlerInput(name = "endpoint", type = String.class, required = true),
+                    @HandlerInput(name = "attrs", type = Map.class, required = true),
+                    @HandlerInput(name = "skipAttrs", type = List.class),
+                    @HandlerInput(name = "onlyUseAttrs", type = List.class),
+                    @HandlerInput(name = "convertToFalse", type = List.class),
+                    @HandlerInput(name = "throwException", type = boolean.class, defaultValue = "true")},
+            output = {
+                    @HandlerOutput(name = "result", type = Map.class)
+            })
+    public static void createEntityResponseOutput(HandlerContext handlerCtx) {
+        Map<String, Object> attrs = (Map) handlerCtx.getInputValue("attrs");
+        if (attrs == null) {
+            attrs = new HashMap<String, Object>();
+        }
+        String endpoint = (String) handlerCtx.getInputValue("endpoint");
+
+        RestResponse response  = sendCreateRequest(endpoint, attrs, (List) handlerCtx.getInputValue("skipAttrs"),
+                (List) handlerCtx.getInputValue("onlyUseAttrs"), (List) handlerCtx.getInputValue("convertToFalse"));
+
+        boolean throwException = (Boolean) handlerCtx.getInputValue("throwException");
+        parseResponse(response, handlerCtx, endpoint, attrs, false, throwException);
+        //??? I believe this should return a Map, whats the point of returning the endpoint that was passed in.
+        //But i haven't looked through all the code, so decide to leave it for now.
+        handlerCtx.setOutputValue("result", response.getResponse());
```

Comment:
Please remove this comment.  This method now returns a map, so the comment doesn't apply.